content/blog/sprint-review-2409.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Wazo Platform 24.09 Released
+date: 2024-07-02T08:00:00
+author: The Wazo Authors
+category: Wazo Platform
+tags: wazo-platform, development
+slug: release-review-2409
+status: published
+---
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 24.09 release.
+
+## New Features in This Release
+
+- **API**: New limit on maximum number of tokens a user can create simultaneously. The limit is set to 100 tokens per user by default, but can be changed in wazo-auth configuration files.
```

Comment:
Do you think that using `session` word will be more intuitive that "tokens a user can create simultaneously"
```suggestion
- **API**: New limit on maximum number of sessions a user can have. The limit is set to 100 sessions per user by default, but can be changed in wazo-auth configuration files.
```

Reply:
Sure, sessions is definitely the appropriate word here, but when talking about `auth` in Wazo i havent been able to find much information about sessions, we often talk about token/refresh tokens

Reply:
Yes I agree, session is not (really) documented
But still no objection for session